WordPressVIPMinimum/Tests/SVG/HTMLCodeSniff.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="299px" height="31px" viewBox="0 0 299 11" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<!-- Generator: Tezt -->
+	<title>Page 100</title>
+	<desc>Created with Sketch.</desc>
+
+
+
+<?php
```

Comment:
I'd like to see some single-line opening and closing tags, and with something in between them, to ensure that the logic handles some likely uses. Just having the opening and closing tags (both types), nicely at the start of different lines, with nothing in between, isn't pushing to find all the edge cases.

Reply:
Thanks! I've added this now :-)

Reply:
Sorry - I wasn't clear - I wanted to see multiple efforts with and without extra content:

```
<?php
?><?php echo 'foo'; ?><?= ?><?php ?><?=

?>
```

etc.